files/validate_settings.rb

```diff
@@ -14,12 +14,14 @@
 # Validate the PE console URL
 begin
   pe_console_url = settings['pe_console_url']
+  cert = `puppet config print localcacert`.chomp
   # The /auth/favicon.ico endpoint is a stable PE console endpoint
   uri = URI.parse("#{pe_console_url}/auth/favicon.ico")
   response = Net::HTTP.start(uri.host,
                              uri.port,
                              use_ssl: uri.scheme == 'https',
-                             verify_mode: OpenSSL::SSL::VERIFY_NONE) do |http|
+                             verify_mode: OpenSSL::SSL::VERIFY_PEER,
```

Comment:
Does it make sense to have the option skip ssl verification here too?

Reply:
I would kind of like to keep it just for the sake of completeness, unless there's some evidence that it causes bugs or something.